syntax = "proto3";
package streetgangpb;


message ErrorResponse
{
    int32 request_id = 1;
	string error_message = 2;
}
